client/src/Components/WorkHistory/WorkHistory.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import "./WorkHistory.css";
+
+function WorkHistory() {
+  return (
+    <div className="WebResume">
+      <div id="List">
+        {/* Skill/Career List */}
+        <div id="CareerGoals">
+          <p2>
+            Highly Driven Individual Seeking An Employment Transition from the
+            Culinary to Software Development Field
+          </p2>
+          <br></br>
+          <br></br>
+          <br></br>
+          <p2 id="pTag">Conflict Resolution and Communication Skills</p2>
+          <br></br>
+          <br></br>
+          <p2>Proven Leadership Ability</p2>
+          <br></br>
+          <br></br>
+          <p2>Developing Skills in JavaScript, HTML, and CSS</p2>
+          <br></br>
+          <br></br>
```

Comment:
Use your CSS for this! having this many `<br>` tags is clunky and can be done much cleaner by using your CSS properties!